data_handler/db/models/zklend_events.py

```diff
@@ -0,0 +1,95 @@
+from decimal import Decimal
+
+from pydantic import BaseModel, ValidationInfo, field_validator
+
+from data_handler.db.models.event import EventBaseModel
+from shared.helpers import add_leading_zeros
+
+
+class AccumulatorsSyncEventData(EventBaseModel):
```

Comment:
It should db model not pydantic model 
you should have fetch fields and types from pydantic model and create db model based on it